community/examples/AMD/hpc-cluster-amd-slurmv5.yaml

```diff
@@ -65,20 +65,46 @@ deployment_groups:
       - type: shell
         source: modules/startup-script/examples/install_ansible.sh
         destination: install_ansible.sh
+      - $(swfs.install_nfs_client_runner)
+      - $(swfs.mount_runner)
       - $(spack.install_spack_deps_runner)
       - $(spack.install_spack_runner)
       # the following installation of AOCC may be automated in the future
```

Comment:
These comments should be in the next block, correct?

Reply:
Yes, I moved it. Thank you.

Reply:
Should be done.